### ISSUE TITLE
Alevin and microwell-seq?

### ISSUE THREAD

**Issue by pinin4fjords** [COMBINE-lab/salmon#358]
Just a question on Alevin (asked on Gitter - but that doesn't seem to be the venue for getting answers ;-) ). 

Has anyone analysed microwell-seq with Alevin? Does it work well?

**Comment by rob-p** [comment]
@k3yavi : thoughts on this?

**Comment by k3yavi** [comment]
Hi @pinin4fjords ,

Apologies for the delayed response and thanks for your interest in Alevin.
Unfortunately, there is no one straight answer for your question. 
Other people have been using Alevin for various microwell based protocols like (CEL-seq https://github.com/COMBINE-lab/salmon/issues/269 ) but from our side we have not extensively tested alevin on non-droplet based protocols. However, we are open to provide any kind of help you may need to test the microwell-seq protocol and extend the support for alevin.

If you happen to have been already testing alevin please let us know of your experience and how we can improve aleivn.

**Comment by pinin4fjords** [comment]
Hi @k3yavi - thanks for the response. We have been looking at using Alevin to support our wider pipelines in the gene expression group at the EBI, as a generic way of quantifying droplet experiments. It has worked well for the 10X v2 studies I've tried, but I'm experiencing some trouble with the drop-seq studies I've tried thus far due to noisy barcodes. May just be bad data, but I'll post an issue or two when I'm clearer.
